test/unit/logic/peers.js

```
@@ -274,9 +275,9 @@ describe('peers', function () {
 			expect(peers.list().length).equal(0);
 		});
 
-		it('should return false when trying to remove non inserted peer', function () {
+		it('should return the error code when trying to remove non inserted peer', function () {
```

`should return an error when trying to remove non-existant peer`